dev-qt/qtnetwork/qtnetwork-5.15.9999.ebuild

```diff
@@ -66,3 +66,10 @@ src_configure() {
 	)
 	qt5-build_src_configure
 }
+
+src_install() {
+	qt5-build_src_install
+	# workaround for bug 652650
+	sed -e "/^#define QT_LINKED_OPENSSL/s/$/ $(usex ssl true false)/" \
```

Comment:
uhm, IIUC this issue, the macro should not be defined at all when `ssl` is disabled.

Reply:
what's in `/usr/include/qt5/QtNetwork/qtnetwork-config.h` if you build with `USE=-ssl`? (not on a gentoo machine right now)

Reply:
essentially what I'm suggesting is

```bash
if use ssl; then
    sed ... '/^#define QT_LINKED_OPENSSL/s/$/ true/' ...
fi
```

Reply:
...right. not much thinking went into this last night. Line is `#define QT_NO_LINKED_OPENSSL` w `USE=-ssl.`